jtr-local.yml

```diff
@@ -18,6 +14,7 @@ flags:
     - 3.3.1
     - 3.2.1
     - 3.1.1
+    - 3.1.1.slim
```

Comment:
Did you mean to throw in 3.1.1-slim?

Reply:
Yes, let me copy a commit description from a commit on this branch where I added this:

Tests: Test on jQuery 3.1.1.slim in non-BrowserStack browser tests

jQuery <3.2.0 doesn't include the deprecated module in the slim build so it
makes sense to test on one of these versions in slim mode even on PRs.

Reply:
...and this was driven by me detecting an issue only after running the full on-branch build and not detecting the issue directly on the PR.